### PR TITLE
Use static examples instead of generting them using faker

### PR DESCRIPTION
**Pull request by alexkwolfe**
Examples need to show all the different supported value formats. So just providing one of the options isn't really a suitable solution.

@cgrayson?

### CODE REVIEW

File: src/boolean.coffee [hunk 1]
```diff
@@ -40,10 +40,13 @@ module.exports =
...
   ],
-  example: ->
-    modifier = [
-      (str) -> str.toUpperCase()
-      (str) -> if str.length > 1 then _.capitalize(str) else str
-      (str) -> str
-    ][_.sample([0, 1, 2])]
-    modifier(_.sample(examples))
+  examples: [
+    'true'
```
Comment (cgrayson) [inline src/boolean.coffee]: Could you use the `examples` array from line 29 here?

Comment (alexkwolfe) [inline src/boolean.coffee]: Oh. oops. I meant to remove that array.

File: src/phone.coffee [hunk 2]
```diff
@@ -149,4 +148,9 @@ module.exports =
     'is included in'
     'is not included in'
   ],
-  example: faker.phone
\ No newline at end of file
+  examples: [
+    '5127891121'
+    '512 789-1121'
```
Comment (cgrayson) [inline src/phone.coffee]: Just asking for trouble (to phone you), but it's your funeral I guess.

Comment (cgrayson) [inline src/phone.coffee]: Could at least use Taylor's number or something :trollface:

**Comment by cgrayson** [comment]
LGR:8ball:TM

**Pull request merged**